pacman-nodejs-app/services/pacman-service.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   type: LoadBalancer
   ports:
     - port: 80
-      targetPort: 80
+      targetPort: 8080
```

Comment:
Indeed, I missed that since I built the image using s2i. I updated the PR on the other repo to update the Dockerfile.